test/debiai-dataprovider-compliance.js

```diff
@@ -78,65 +113,73 @@ describe('Testing the debiai dataprovider compliance', function () {
-
                 idsReturned.forEach(id => {
-                    if (dataReturned[id].length !== providerViewsColumns[view].length)
-                        throw new Error(`Columns and data length mismatch for view ${view} (${providerViewsColumns[view].length} was expected but got ${dataReturned[id].length}) at data with id ${id}`);
+                    if (dataReturned[id].length !== providerProjectsColumns[projectId].length)
+                        throw new Error(`Columns and data length mismatch for project ${projectId} (${providerProjectsColumns[projectId].length} was expected but got ${dataReturned[id].length}) at data with id ${id}`);
                 });
             }
         }
     })
 
     // Models
     it('should expose some models', async () => {
-        for (let viewNb = 0; viewNb < providerViews.length; viewNb++) {
-            const view = providerViews[viewNb];
+        for (let projectNb = 0; projectNb < providerProjects.length; projectNb++) {
+            const projectId = providerProjects[projectNb];
 
             let resp = await request(app)
-                .get(`/debiai/view/${view}/models`)
+                .get(`/debiai/projects/${projectId}/models`)
                 .set('Content-Type', 'application/json')
                 .expect(200)
                 .expect('Content-Type', /json/)
 
-            viewsModels[view] = resp.body;
+            projectsModels[projectId] = resp.body;
         }
     });
 
     it('should expose the models evaluated data id list', async () => {
-        for (let viewNb = 0; viewNb < providerViews.length; viewNb++) {
-            const view = providerViews[viewNb];
+        for (let projectNb = 0; projectNb < providerProjects.length; projectNb++) {
+            const projectId = providerProjects[projectNb];
 
-            viewsModelsEvaluatedDataIdList[view] = {}
-            for (let modelNb = 0; modelNb < viewsModels[view].length; modelNb++) {
-                const model = viewsModels[view][modelNb];
+            projectsModelsEvaluatedDataIdList[projectId] = {}
+            for (let modelNb = 0; modelNb < projectsModels[projectId].length; modelNb++) {
+                const model = projectsModels[projectId][modelNb];
 
                 let resp = await request(app)
-                    .get(`/debiai/view/${view}/model/${model.id}/evaluatedDataIdList`)
+                    .get(`/debiai/projects/${projectId}/models/${model.id}/evaluated-data-id-list`)
                     .set('Content-Type', 'application/json')
                     .expect(200)
                     .expect('Content-Type', /json/)
 
-                viewsModelsEvaluatedDataIdList[view][model.id] = resp.body;
+                projectsModelsEvaluatedDataIdList[projectId][model.id] = resp.body;
             }
         }
     });
 
+    it('should return a 404 error : Model not found', (done) => {
+        const projectId = "project_1";
+        const modelId = "model_10"
```

Comment:
Change to something like "model_that_doesnt_exist"
the probability that someone has a model_x is to high